app/controllers/api_controller/tags.rb

```diff
@@ -79,7 +79,7 @@ def edit_resource_tags(type, id, data)
     def delete_resource_tags(_type, id, _data = {})
       destroy_tag_and_classification(id)
       action_result(true, "tags id: #{id} deleting")
-    rescue => err
+    rescue BadRequestError => err
```

Comment:
Ah, didn't fully understand this then. Could be tricky....best I can think of is to re-raise here if it's `ActiveRecord::RecordNotFound` - is that less gross than failing on it as I was before?